src/landing/index.js

```diff
@@ -25,7 +25,7 @@ function Landing () {
       <NavBar ref={navBarRef} variant={navBarVariant} />
       <article className={classes.article}>
         <Header ref={headerRef} />
-        <LetsTalk />
+        <ContactForm />
       </article>
```

Comment:
Did you run `git pull origin develop` before creating your new branch? This code doesn't exist anymore.